tools/licenses/lib/main.dart

```diff
@@ -1993,13 +1993,40 @@ class RepositoryBoringSSLSourceDirectory extends RepositoryDirectory {
   }
 
   @override
+  RepositoryFile createFile(fs.IoNode entry) {
+    if (entry.name == 'LICENSE')
+      return new RepositoryOpenSSLLicenseFile(this, entry);
+    return super.createFile(entry);
+  }
+
+  @override
   RepositoryDirectory createSubdirectory(fs.Directory entry) {
     if (entry.name == 'third_party')
       return new RepositoryBoringSSLThirdPartyDirectory(this, entry);
     return super.createSubdirectory(entry);
   }
 }
 
+class RepositoryOpenSSLLicenseFile extends RepositorySingleLicenseFile {
+  RepositoryOpenSSLLicenseFile(RepositoryDirectory parent, fs.TextFile io)
+    : super(parent, io, new License.message(io.readString(), LicenseType.openssl, origin: io.fullName)) {
```

Comment:
This is the place where we should trim off the first N lines (whatever number of lines the lawyer said to trim off)